sdk/metricsadvisor/ai-metrics-advisor/recordings/browsers/aad_metricsadvisoradministrationclient_datafeed_datafeed/recording_creates_an_azure_application_insights_feed.json

```diff
@@ -11,53 +11,53 @@
+   "response": "{\"dataFeedId\":\"1e98bb5f-3c7d-43ca-969f-775150de3ada\",\"dataFeedName\":\"js-test-appInsightsFeed-163642031976602855\",\"metrics\":[{\"metricId\":\"504070b0-2743-4c17-b917-fa33217ee2ed\",\"metricName\":\"cost\",\"metricDisplayName\":\"cost\",\"metricDescription\":\"\"},{\"metricId\":\"79c7752a-646d-497f-95ca-582825652d27\",\"metricName\":\"revenue\",\"metricDisplayName\":\"revenue\",\"metricDescription\":\"\"}],\"dimension\":[{\"dimensionName\":\"category\",\"dimensionDisplayName\":\"category\"},{\"dimensionName\":\"city\",\"dimensionDisplayName\":\"city\"}],\"dataStartFrom\":\"2020-08-21T00:00:00Z\",\"dataSourceType\":\"AzureApplicationInsights\",\"timestampColumn\":\"\",\"startOffsetInSeconds\":0,\"maxQueryPerMinute\":30.0,\"granularityName\":\"Daily\",\"allUpIdentification\":\"__CUSTOM_SUM__\",\"needRollup\":\"NeedRollup\",\"fillMissingPointType\":\"CustomValue\",\"fillMissingPointValue\":555.0,\"rollUpMethod\":\"Sum\",\"dataFeedDescription\":\"Data feed description\",\"stopRetryAfterInSeconds\":-1,\"minRetryIntervalInSeconds\":-1,\"maxConcurrency\":-1,\"viewMode\":\"Private\",\"admins\":[\"azure_client_id\"],\"viewers\":[],\"creator\":\"azure_client_id\",\"status\":\"Active\",\"createdTime\":\"2021-11-09T01:12:03Z\",\"isAdmin\":true,\"actionLinkTemplate\":\"\",\"dataSourceParameter\":{\"query\":\"let gran=60m; let starttime=datetime(@StartTime); let endtime=starttime + gran; requests | where timestamp >= starttime and timestamp < endtime | summarize request_count = count(), duration_avg_ms = avg(duration), duration_95th_ms = percentile(duration, 95), duration_max_ms = max(duration) by resultCode\",\"azureCloud\":\"Azure\",\"applicationId\":\"appInsights_application\"},\"authenticationType\":\"Basic\"}",
    "responseHeaders": {
-    "apim-request-id": "e221dee7-f93a-4075-9867-9883f67fdd13",
+    "apim-request-id": "3b3c07e9-056d-4fe7-b250-bf55f100939b",
     "content-length": "1624",
     "content-type": "application/json; charset=utf-8",
-    "date": "Mon, 08 Nov 2021 09:44:25 GMT",
+    "date": "Tue, 09 Nov 2021 01:12:03 GMT",
     "strict-transport-security": "max-age=31536000; includeSubDomains; preload",
     "x-content-type-options": "nosniff",
-    "x-envoy-upstream-service-time": "108",
-    "x-request-id": "e221dee7-f93a-4075-9867-9883f67fdd13"
+    "x-envoy-upstream-service-time": "122",
+    "x-request-id": "3b3c07e9-056d-4fe7-b250-bf55f100939b"
```

Comment:
Why did this recording need to change? I don’t see anything that needed to change in the entirety of this file.